core/src/main/scala/akka/kafka/Metadata.scala

```diff
@@ -170,7 +170,7 @@ object Metadata {
    * [[org.apache.kafka.clients.consumer.KafkaConsumer#committed()]]
    */
   final case class GetCommittedOffset(partition: TopicPartition) extends Request with NoSerializationVerificationNeeded
-  final case class CommittedOffset(response: Try[OffsetAndMetadata])
+  final case class CommittedOffset(response: Try[OffsetAndMetadata], requestedPartition: TopicPartition)
```

Comment:
I believe it's more natural to have 
```suggestion
  final case class CommittedOffset(partition: TopicPartition, response: Try[OffsetAndMetadata])
```

Reply:
Personally I'd expect the other way around, the response to my query as the first argument and a reminder of what my query was second (i.e. the thing I definitely want first and the thing I might also want second). Either way round is fine imo though.